vars/runPipeline.groovy

```diff
@@ -328,7 +347,13 @@ DFLAGS=-I%@P%/../imports -L-L%@P%/../libs -L--export-dynamic -L--export-dynamic
 
     def dub_projects = [
         // sorted by test time slow to fast
-        "rejectedsoftware/vibe.d",
+        "vibe-d/vibe.d+libevent-base",
+        "vibe-d/vibe.d+libevent-examples",
+        "vibe-d/vibe.d+libevent-tests",
```

Comment:
Vibe-core is the new default with 0.8.3, so it might be worth a consideration to drop these tests as libevent is on its way out.
After all, they are really expensive and the goal of the project tester is to avoid that we break projects, not add another CI for them.
See: https://github.com/vibe-d/vibe.d/pull/2002

Reply:
It's quite an important project, so it seems better we cover all the supported code.